templates/examples/cryptocurrency-slack-bot/index.js

```diff
@@ -1,12 +1,8 @@
-addEventListener("fetch", (event) => {
-	event.respondWith(slackWebhookHandler(event.request));
-});
-
 // SLACK_TOKEN is used to authenticate requests are from Slack.
 // Keep this value secret.
 let SLACK_TOKEN = "PUTYOURTOKENHERE";
```

Comment:
Slightly unrelated to this PR, but we probably should not encourage embedding secrets within source code. We ought to refactor this template to depend on a Secret env var.